src/styles/blocks/card.scss

```
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    width: 166px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16px 4px;
```

reset default margin for all `p` and `h2`. And use only margin-bottom for card children 
```suggestion
    margin-bottom: 4px;
```